tempel.el

```diff
@@ -48,9 +48,11 @@
   :group 'editing
   :prefix "tempel-")
 
-(defcustom tempel-file (expand-file-name "templates" user-emacs-directory)
-  "Path to the template file."
-  :type 'string)
+(defcustom tempel-template-locations nil
```

Comment:
I'll do a very quick first pass review. Please call this variable `tempel-paths`. I prefer concise names.